### PR TITLE
Added PR and Issue template.

### PR DESCRIPTION
**Pull request by gyliu513**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/88)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
```
Comment (ellis-bigelow) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Istio Version
Knative Version
KFServing Version

**Comment by ellis-bigelow** [comment]
Thanks Guang!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide/pull-requests.md#the-pull-request-submit-process and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
```
Comment (yuzisun) [inline .github/PULL_REQUEST_TEMPLATE.md]: should we use https://www.kubeflow.org/docs/about/contributing/ and our development guide here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
+4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+5. If this PR changes image versions, please title this PR "Bump <image name> from x.x.x to y.y.y."
```
Comment (yuzisun) [inline .github/PULL_REQUEST_TEMPLATE.md]: I am not sure  if 5. applies to our case.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (gyliu513) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (gyliu513) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (gyliu513) [reply .github/PULL_REQUEST_TEMPLATE.md]: Deleted

**Comment by ellis-bigelow** [comment]
/lgtm
@yuzisun, everything look good to you?

**Comment by yuzisun** [comment]
ye looks good, thanks!
/approve

**Pull request merged**